### ISSUE TITLE
Column p.proisagg does not exist

### ISSUE THREAD

**Issue by gomezjdaniel** [dbcli/pgcli#1000]
## Description
So after getting a fresh installation of a PostgreSQL server and login into with pgcli I get the following error:

```
➜ pgcli -h localhost -p 5432 -U postgres -W postgres -d postgres
Password: 
Exception in thread completion_refresh:
Traceback (most recent call last):
  File "/usr/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.5/threading.py", line 862, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/share/pgcli/pgcli/completion_refresher.py", line 58, in _bg_refresh
    refresher(completer, executor)
  File "/usr/share/pgcli/pgcli/completion_refresher.py", line 100, in refresh_functions
    completer.extend_functions(executor.functions())
  File "/usr/share/pgcli/pgcli/pgcompleter.py", line 133, in extend_functions
    for f in func_data:
  File "/usr/share/pgcli/pgcli/pgexecute.py", line 384, in functions
    cur.execute(self.functions_query)
psycopg2.ProgrammingError: column p.proisagg does not exist
LINE 6:                 p.proisagg is_aggregate,
                        ^
HINT:  Perhaps you meant to reference the column "p.prolang".
```
Nonetheless this error I can keep using pgcli.

## Your environment

Ubuntu 16.04.5 LTS

```
pgcli -v    
Version: 0.20.1
```
```
postgres> select version();
+----------------------------------------------------------------------------------------------------------------------------------+
| version                                                                                                                          |
|----------------------------------------------------------------------------------------------------------------------------------|
| PostgreSQL 11.1 (Debian 11.1-1.pgdg90+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 6.3.0-18+deb9u1) 6.3.0 20170516, 64-bit |
+----------------------------------------------------------------------------------------------------------------------------------+
```

**Comment by j-bennet** [comment]
It appears we should use `pg_proc.prokind` in Postgresql 11:

https://git.postgresql.org/gitweb/?p=postgresql.git;a=commit;h=fd1a421fe66173fb9b85d3fe150afde8e812cbe4

**Comment by gomezjdaniel** [comment]
Which apparently was fixed here already https://github.com/dbcli/pgcli/issues/919 :thinking:

**Comment by gomezjdaniel** [comment]
Fixed,

I had installed pgcli via apt instead of using pip so I was using and old version (`0.20.1`). So removed it and installed again via pip (`2.0.2`)

**Comment by j-bennet** [comment]
👍

**Comment by Abhinickz** [comment]
Getting the same error:
```
[16:16:35 abhinickz@abhinickz -> ~$ pgcli test
Version: 1.9.1
Chat: https://gitter.im/dbcli/pgcli
Mail: https://groups.google.com/forum/#!forum/pgcli
Home: http://pgcli.com
test> Exception in thread completion_refresh:
Traceback (most recent call last):
  File "/usr/lib/python3.7/threading.py", line 917, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.7/threading.py", line 865, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/share/pgcli/pgcli/completion_refresher.py", line 68, in _bg_refresh
    refresher(completer, executor)
  File "/usr/share/pgcli/pgcli/completion_refresher.py", line 146, in refresh_functions
    completer.extend_functions(executor.functions())
  File "/usr/share/pgcli/pgcli/pgcompleter.py", line 224, in extend_functions
    for f in func_data:
  File "/usr/share/pgcli/pgcli/pgexecute.py", line 612, in functions
    cur.execute(query)
psycopg2.ProgrammingError: column p.proisagg does not exist
LINE 8:                         p.proisagg is_aggregate,
                                ^
HINT:  Perhaps you meant to reference the column "p.prolang".
```
```
[16:17:30 abhinickz@abhinickz -> ~$ pgcli --version
Version: 1.9.1
```
```
[16:17:33 abhinickz@abhinickz -> ~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 19.04
Release:	19.04
Codename:	disco
```
```
[16:18:02 abhinickz@abhinickz -> ~$ psql --version
psql (PostgreSQL) 11.3 (Ubuntu 11.3-1.pgdg18.10+1)
```

**Comment by j-bennet** [comment]
@Abhinickz Try updating: `pip install -U pgcli`. Your version is 1.9.1 and the fix was added in 2.0.2.

**Comment by aesyondu** [comment]
> @Abhinickz Try updating: `pip install -U pgcli`. Your version is 1.9.1 and the fix was added in 2.0.2.

Confirming that this works. I am using Ubuntu 18.04 via WSL2, and when I `sudo apt install pgcli` the version is 1.6. Using `sudo apt install python3-pip` and `pip3 install pgcli` the installed version is now 3.1.0.

**Comment by Abhinickz** [comment]
> @Abhinickz Try updating: `pip install -U pgcli`. Your version is 1.9.1 and the fix was added in 2.0.2.

It works now.
